dnd_app/app.py

```diff
@@ -0,0 +1,28 @@
+# import the Flask class from the flask module
+from flask import Flask, render_template, redirect, url_for, request
+
+# create the application object
+app = Flask(__name__)
+
+# use decorators to link the function to a url
+@app.route('/')
+def home():
+    return "Hello, World!"  # return a string
+
+@app.route('/welcome')
```

Comment:
you seem to be redoing some work that already exists. if you look in the routes.py file you'll see we already have most of the routing done and a welcome page (called index)